src/components/select-project/select-project.controller.ts

```diff
@@ -73,7 +73,9 @@ export class SelectProjectController implements angular.IController {
       this.ctrl.existingProjectNames = _.map(this.ctrl.projects, 'metadata.name');
 
       // get most recently created
-      this.ctrl.selectedProject = this.$filter('mostRecent')(this.ctrl.projects);
+      if (!this.ctrl.selectedProject) {
```

Comment:
Just confirming that this selects the correct project since the outside object reference won't match the one that comes back in the DataService.list call from inside this component.